AWSWindowsHelpers/public/ARN/Get-AWSWindowsHelpersARNDetail.ps1

```diff
@@ -0,0 +1,128 @@
+<#
+.Synopsis
+   Returns the details of an AWS Resource from the ARN
+.DESCRIPTION
+   Returns the details of an AWS Resource from the ARN
+.EXAMPLE
+   Get-AWSWindowsHelpersARNDetail -arn "arn:aws:iam::123456789012:server-certificate/2017_wild_example_com"
+.INPUTS
+   arn - ARN of the AWS certificate
+.OUTPUTS
+   Returns a object with the following properties
```

Comment:
It would help in these comments/help if you gave an example output, as I'm a bit confused as to what this does reading it!